### PR TITLE
Story1 show collection

### PR DESCRIPTION
**Pull request by henridvf**
Review and pull request to trainer Neal!

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+}
+
+.heading .footer {
+    justify-content: start;
+}
+
+.collection_item {
+    color: #000000;
+    margin: 5px 5px;
+    background-color: lightgrey;
```
Comment (nealioa) [inline css/styles.css:35]: The use of a named colour is inconsistent. I suggest using hash values everywhere.

File: functions.php [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+}
+
+/**
+ * Helper function to get and format the image path
+ * @param string $imagePath
+ * @return string
+ */
+function getImagePath(string $imagePath): string
+{
+    $path='';
```
Comment (nealioa) [inline functions.php:27]: A space either side of `=` is nicer.

File: functions.php [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+ */
+function displayCollection(): string
+{
+    $display = '';
+    $watches = getAllWatches();
+
+    foreach ($watches as $watch) {
+        // create and add to display collection
+        $display .= createCollectionItem($watch);
+    };
```
Comment (nealioa) [inline functions.php:51]: A semicolon after a curly bracket is not needed.

**Review by nealioa — CHANGES_REQUESTED** [review]

File: css/styles.css (hunk 1 above)
Comment (henridvf) [reply css/styles.css:35]: Changed that to its hash value.

File: functions.php (hunk 2 above)
Comment (henridvf) [reply functions.php:27]: Corrected

File: functions.php (hunk 3 above)
Comment (henridvf) [reply functions.php:51]: Corrected

**Pull request merged**